src/assets/scss/elements/ys-flow-steps.scss

```diff
@@ -4,22 +4,32 @@
 @import '../tools/ys-functions';
 
 .ys-flow-steps {
+  $variant-vertical-ref: 0;
+  $bullet-size-px: 34;
   color: $ys-color-grey-28;
 
+  &--vertical {
+    $variant-vertical-ref: &;

```

Comment:
Using a number then overwriting with this should not be done - the `&` will outside a class provide a false value (hence you can set the value outside to the same input and receive the same result) then set it to this value as a "cleaner" alternative (see top example for reference in the docs):

https://sass-lang.com/documentation/values/null

Reply:
I’m not sure about this. Why are you creating modifiers like this? The code is more difficult to read IMO 😊